.github/README.md

```diff
@@ -1,164 +1,74 @@
+  * We strive to make all in-game content to be blizzlike. Therefore we have a high standard for fixes being made.
 
+* Customization
+  * It should be easy to customize your experience using [modules](#modules).
 
-If you want to contribute to the project, you will find a lot of resources that will guide you in our wiki.
+* Community driven
+  * AzerothCore has an active community of developers, contributors, and users who collaborate, share knowledge, and provide support through forums, Discord channels, and other communication platforms 
 
-Feel free to join us on [our Discord chat server](https://discord.gg/gkt4y2x) where we teach a lot of new people how to get started and who are now important contributors!
+### Modules
 
-<!-- TO UNCOMMENT LATER -->
-<!-- As we put a big emphasis on community, there are also special rewards for contributors such as reputation ranks (displayed in our Discord and our website), reputation badges (to display on your project/portfolio), premium software licenses, private modules access, private tools access and a lot of small private repositories access. -->
+AzerothCore is designed to be highly modular, allowing developers to extend and customize the game to suit their preferences or create unique gameplay experiences. This flexibility enables the addition of custom features, content, and modifications.
 
+We have a lot of modules already made by the community which can be found in the [Module Catalouge](https://www.azerothcore.org/catalogue.html#/).
 
 ## Installation
 
-Installation instructions are available [here](http://www.azerothcore.org/wiki/Installation).
+Detailed installation instructions are available [here](http://www.azerothcore.org/wiki/installation).
 
-We also have an auto-installation bash script [here](/apps/installer/main.sh) (*Warning: try it/analyze it before running it*).
-
-Dockerization of AzerothCore is fully supported, and we have various community-made tutorials (eg: AWS / Digital Ocean installation).
+## Contributing
 
+AzerothCore can also serve as a learning resource for aspiring developers who want to understand how WoW servers work, how MMORPGs are structured, how game server emulators are created or to improve their C++ and SQL knowledge.
 
-## Support
+If you want to contribute to the project, you will find a lot of resources that will guide you in our [wiki](https://www.azerothcore.org/wiki/contribute).
 
-Our self-made wiki probably has a lot of answers for you.
+We also recommend you read our [CODE OF CONDUCT](https://github.com/azerothcore/azerothcore-wotlk/blob/master/.github/CODE_OF_CONDUCT.md).
 
-For help requests, it is recommended to ask your question on [StackOverflow](https://stackoverflow.com/questions/tagged/azerothcore) and link it in [our chat](https://discordapp.com/channels/217589275766685707/284406375495368704).
+Feel free to join our [Discord server](https://discord.gg/gkt4y2x)
 
+Click on the "⭐ Star" button to help us gain more visibility on Github!
```

Comment:
Might look better using backticks (`) instead of double quotation marks (").

Reply:
that defeats the emoji